readme.md

```diff
@@ -24,7 +24,7 @@ Synchronize Ewon Flexy data to Ignition's Tag Historian.
 
 ## [Version](#table-of-contents)
 
-This module version is compatible with Ignition 7.9.  For an Ignition 8 compatible module go to [Ignition-8-master](https://github.com/hms-networks/IgnitionEwonConnector/tree/Ignition-8-master) branch.
+This module version is compatible with Ignition 7.9, but is no longer supported and does not receive updates. For an Ignition 8 compatible module go to [main](https://github.com/hms-networks/IgnitionEwonConnector/tree/main) branch.
```

Comment:
```suggestion
This module version is compatible with Ignition 7.9. For an Ignition 8 compatible module go to [main](https://github.com/hms-networks/IgnitionEwonConnector/tree/main) branch.

Note: The Ignition 7.9 Ewon module is no longer supported and this version will not receive future updates.
```

Reply:
I think including the word but makes it sound like someone shouldnt use it.  We might even go as far to phrase it "Note: The Ignition 7.9 Ewon module will not receive future updates." just to not discourage someone from using it if they have 7.9 deployed.

Reply:
I agree.